redis.go

```diff
@@ -217,7 +217,7 @@ func redisRange(l, start, end int, stringSymantics bool) (int, int) {
 			}
 		}
 	}
-	if end < math.MaxInt {
+	if end < math.MinInt32 {
```

Comment:
Any reason you went for _min_?

Reply:
Ah, that's a mistake done in haste. Let me fix this.